src/groovy/com/unifina/domain/CustomStreamIDValidator.groovy

```diff
@@ -0,0 +1,23 @@
+package com.unifina.domain
+
+import grails.compiler.GrailsCompileStatic
+
+@GrailsCompileStatic
+class CustomStreamIDValidator {
+
+	public static final String SANDBOX_PREFIX = "sandbox"
+	public static final String PATH_REGEX = "^/(?:[\\w\\.-]+/?)*\\w\$"
```

Comment:
Could use an explanatory comment for better readabiity. In natural language, what are the rules?